packages/react/src/constants/device-sizes.constant.ts

```diff
@@ -36,3 +38,39 @@ export const sizes = {
     return 'large';
   },
 };
+
+interface Breakpoints {
+  small: number;
+  medium: number;
+}
+
+export const getSizesForBreakpoints = ({ small, medium }: Breakpoints) => {
```

Comment:
couldn't do this since I had to use typeof sizes to address comment in 
<img width="810" alt="image" src="https://user-images.githubusercontent.com/98028693/190951365-13eeef11-b834-426a-ac7e-6d623781b9a4.png">

Reply:
You can do so by only exporting the type:

```ts
const sizes = //...

export const Sizes = typeof sizes
```

Reply:
done